src/pages/InstitutionSettings/InstitutionSettings.tsx

```diff
@@ -30,7 +31,15 @@ const InstitutionSettings: FC = () => {
       </div>
       <Container className={classes.container}>
         <h3 className={classes.title}>{t('institution.institution_data')}</h3>
-        <InstitutionForm name={name} id={institutionId} {...institution} />
+        <InstitutionForm
+          name={name}
+          id={institutionId}
+          {...institution}
+          workingTimes={
+            <WorkingTimes name={name} id={institutionId} data={institution} />
+          }
+          //vacationDays={<div>Puhkepäevad:</div>}
```

Comment:
Commented out, left in

Reply:
This is needed. It's TODO, I will add TODO comment as well